tests/data/StoreLibraries.json

```diff
@@ -11,7 +11,6 @@
     "created_at": "09/23/2019 11:18",
     "deactivated_at": null,
     "sample_names": "4616STDY7535900,4616STDY7535900",
-    "tag_oligos": "ATGC,CGTA",
     "tag_group_ids": "1"
```

Comment:
Here tag_group_ids would be "1,2" I guess?

Reply:
yep, good spot